i18n/de.json

```diff
@@ -1,23 +1,30 @@
 {
 	"@metadata": {
 		"authors": [
+			"0x010C",
 			"Ameisenigel",
 			"Anna7321",
 			"Brettchenweber",
 			"CyberOne25",
 			"FF-11",
 			"FF11",
+			"Lorem Ipsum",
 			"Metalhead64",
 			"Sebastian Wallroth",
+			"WikiLucas00",
 			"TMg",
 			"ThisCarthing",
 			"Zabe"
+
 		]
 	},
 	"recordWizard": "Aufnahmeassistent",
 	"recordwizard": "Aufnahmeassistent",
 	"recordWizard-desc": "Ein benutzerfreundliches und automatisiertes Werkzeug zur Aufnahme von Aussprache-Audiodateien.",
 	"special-recordWizard-title": "Aufnahmeassistent",
+	"mwe-recwiz-menu-statistics": "Statistiken",
+	"mwe-recwiz-menu-datasets": "Datensätze",
+	"mwe-recwiz-menutitle": "Menü",
```

Comment:
No such message key in en.json. Translations will be removed in next export.

Reply:
Thanks for checking. It's not a problem. It's an old pull request that was finally merged at the hackathon. I'll probably be doing some more cleanups soon.